workflow/rocoto/gefs_tasks.py

```diff
@@ -483,3 +483,48 @@ def wavepostpnt(self):
         task = rocoto.create_task(member_metatask_dict)
 
         return task
+
+    def extractvars(self):
+        deps = []
+        if self.app_config.do_wave:
+            dep_dict = {'type': 'task', 'name': f'wave_post_grid_mem#member#'}
```

Comment:
```suggestion
            dep_dict = {'type': 'task', 'name': 'wave_post_grid_mem#member#'}
```
There is no `f-string` involved here.